### PR TITLE
[PLAY-21] adding-opacity-in-playbook-review

### PR DESCRIPTION
**Pull request by cvargaspower**
#### Breaking Changes
Need to change the opacity order correctly. Right now its 1 being transparent when 10 should be transparent. 

#### Process Changes
- Used the data_1 with primary bg-color to create the display/look I wanted. 
- Added an example Import in SCSS (couldn't figure out how to make it dark bg like the example) 
- Added a table with all the variables and code

#### Runway Ticket URL

[ [PLAY 21] ](https://nitro.powerhrg.com/runway/backlog_items/PLAY-21)

#### How to test this

[INSERT TESTING DETAILS]

#### Checklist:

- [x] **LABELS** Add a label: `enhancement`, `bug`, `improvement`, `new kit`, `deprecated`, or `breaking`. See [Changelog & Labels](https://github.com/powerhome/playbook/wiki/Changelog-&-Labels) for details.
- [x] **DEPLOY** Please add the `Milano` label when you are ready for a review.
- [x] **SCREENSHOT** Please add a screen shot or two.
- [x] **SPECS** Please cover your changes with specs.
- [x] **READ DOCS** Please make sure you have read and understand the [Playbook Release Process](https://github.com/powerhome/playbook/wiki/Playbook-Releases)

### CODE REVIEW

**Comment by jasoncypret** [comment]
@cvargaspower I'm going to give you some updates on the ticket...but here's some code you may need. https://share.getcloudapp.com/p9uNLD4Z

**Review by jasoncypret — CHANGES_REQUESTED** [review]
Great job. Just some additional feedback for you. I worked up a quick screenshot with direct annotations below. Please address these:



You this mock and the one attached to the story as a guide:


Here’s this code: [example.erb](https://share.getcloudapp.com/9ZuRKBQe)

**Comment by jasoncypret** [comment]
@cvargaspower 

1. Can you add the correct label? See the guidelines for that.
2. Also, can you start your story with the runway number in brackets. ex [PLAY-21]
3. Can you make your title more clear? These titles are used for our changelog. What are you adding/improving?

File: playbook-website/app/javascript/site_styles/_visual_guidelines.scss [hunk 1]
```diff
@@ -42,4 +44,35 @@
...
+  }
+  .opacity_8{
+    opacity: $opacity_8;
+  }
+  .opacity_9{
+    opacity: $opacity_9;
+  }
+  .opacity_10{
+    opacity: $opacity_10;
+  }
```
[the remark this answers is not in the document]
Comment (cvargaspower) [reply playbook-website/app/javascript/site_styles/_visual_guidelines.scss:77]: For my feedback, can you explain what you mean the 10, 20, etc? Did you want me to change the opacity to those numbers starting from 10?

File: playbook-website/app/views/pages/visual_guidelines.html.erb [hunk 2]
```diff
@@ -14,9 +14,12 @@
...
   <%= pb_rails("background", props: { classname: "token-wrapper", background_color: "light", padding: "xl" }) do %>
       <%= render "pages/visual_guidelines_partials/colors.html.slim" %>
+      <%= render "pages/visual_guidelines_partials/opacity.html.erb" %>
       <%= render "pages/visual_guidelines_partials/max_width.html.slim" %>
       <%= render "pages/visual_guidelines_partials/number_spacing.html.slim" %>
       <%= render "pages/visual_guidelines_partials/positioning.html.slim" %>
       <%= render "pages/visual_guidelines_partials/spacing.html.slim" %>
   <% end %>
+
+
```
Comment (benlangfeld) [inline playbook-website/app/views/pages/visual_guidelines.html.erb:24]:
```suggestion
```

File: playbook-website/app/javascript/site_styles/_visual_guidelines.scss [hunk 3]
```diff
@@ -42,4 +44,6 @@
     .nine { margin-left: 400px; }
     .ten { margin-left: 450px; }
   }
+
+
```
Comment (benlangfeld) [inline playbook-website/app/javascript/site_styles/_visual_guidelines.scss:48]:
```suggestion
```

File: playbook-website/app/javascript/site_styles/_opacity_guidelines.scss [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+    opacity: $opacity_8;
+  }
+  .opacity_9{
+    opacity: $opacity_9;
+  }
+  .opacity_10{
+    opacity: $opacity_10;
+  }
+
+}
```
Comment (benlangfeld) [inline playbook-website/app/javascript/site_styles/_opacity_guidelines.scss:35]:
```suggestion
}

```

File: playbook-website/app/javascript/site_styles/_visual_guidelines.scss [hunk 5]
```diff
@@ -42,4 +44,6 @@
     .nine { margin-left: 400px; }
     .ten { margin-left: 450px; }
   }
+
+ 
 }
```
Comment (benlangfeld) [inline playbook-website/app/javascript/site_styles/_visual_guidelines.scss:49]:
```suggestion
}

```

**Comment by cvargaspower** [comment]
#### Screens

 

#### Breaking Changes
- Need to change the opacity order correctly. Right now its 1 being transparent when 10 should be transparent. 

#### Fixed Changes
- added a separate _opacity_guideliness.scss within the style-css folder, BUT it wouldn't connect after import so I reverted back to what the code was before the change. The file is still located within the PR, but just wanted to make that note. 
- Added a description
- Added a description on how to use Tokens
- Added a title so users know where they are: i.e opacity tokens
- Added the correct highlight coded sample.

#### Runway Ticket URL

[ [PLAY 21] ](https://nitro.powerhrg.com/runway/backlog_items/PLAY-21)

#### How to test this

[INSERT TESTING DETAILS]

#### Checklist:

- [x] **LABELS** Add a label: `enhancement`, `bug`, `improvement`, `new kit`, `deprecated`, or `breaking`. See [Changelog & Labels](https://github.com/powerhome/playbook/wiki/Changelog-&-Labels) for details.
- [x] **DEPLOY** Please add the `Milano` label when you are ready for a review.
- [x] **SCREENSHOT** Please add a screen shot or two.
- [x] **SPECS** Please cover your changes with specs.
- [x] **READ DOCS** Please make sure you have read and understand the [Playbook Release Process](https://github.com/powerhome/playbook/wiki/Playbook-Releases)

File: playbook-website/app/javascript/site_styles/_visual_guidelines.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (jasoncypret) [reply playbook-website/app/javascript/site_styles/_visual_guidelines.scss:78]: For `$opacity_1` what is the value of that? for `$opacity_2` what is the value of that? Display those values there

File: playbook-website/app/javascript/site_styles/_opacity_guidelines.scss [hunk 6]
```diff
@@ -0,0 +1,38 @@
+
+
+
```
Comment (benlangfeld) [inline playbook-website/app/javascript/site_styles/_opacity_guidelines.scss:3]:
```suggestion
```

File: playbook-website/app/javascript/site_styles/_opacity_guidelines.scss [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+    opacity: $opacity_8;
+  }
+  .opacity_9{
+    opacity: $opacity_9;
+  }
+  .opacity_10{
+    opacity: $opacity_10;
+  }
+  
+
```
Comment (benlangfeld) [inline playbook-website/app/javascript/site_styles/_opacity_guidelines.scss:36]:
```suggestion
```

File: playbook-website/app/javascript/site_styles/_opacity_guidelines.scss [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+
+
+.selector { 
+    .opacity_1{
```
Comment (benlangfeld) [inline playbook-website/app/javascript/site_styles/_opacity_guidelines.scss:5]:
```suggestion
  .opacity_1{
```

File: playbook-website/app/javascript/site_styles/_opacity_guidelines.scss [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+  .opacity_9{
+    opacity: $opacity_9;
+  }
+  .opacity_10{
+    opacity: $opacity_10;
+  }
+  
+
+}
+
```
Comment (benlangfeld) [inline playbook-website/app/javascript/site_styles/_opacity_guidelines.scss:38]:
```suggestion
```

**Comment by cvargaspower** [comment]
@benlangfeld I pushed another/committed with the space changes, hopefully all the coding spaces are good to go. Sorry about that... it must have happened when I was copy/pasting/moving things around.

**Comment by kre8sions** [comment]
@cvargaspower  lets try to make the example look like this with the dark area and the card connected

Screen Shot 2021-11-18 at 12 54 46 PM

**Comment by cvargaspower** [comment]
#### Screens

 

#### Breaking Changes
- Need to change the opacity order correctly. Right now its 1 being transparent when 10 should be transparent. 

#### Fixed Changes
- Whitespaces have been fixed and space with dark card and light card also fixed!
- added a separate _opacity_guideliness.scss within the style-css folder, BUT it wouldn't connect after import so I reverted back to what the code was before the change. The file is still located within the PR, but just wanted to make that note. 
- Added a description
- Added a description on how to use Tokens
- Added a title so users know where they are: i.e opacity tokens
- Added the correct highlight coded sample.

#### Runway Ticket URL

[ [PLAY 21] ](https://nitro.powerhrg.com/runway/backlog_items/PLAY-21)

#### How to test this

[INSERT TESTING DETAILS]

#### Checklist:

- [x] **LABELS** Add a label: `enhancement`, `bug`, `improvement`, `new kit`, `deprecated`, or `breaking`. See [Changelog & Labels](https://github.com/powerhome/playbook/wiki/Changelog-&-Labels) for details.
- [x] **DEPLOY** Please add the `Milano` label when you are ready for a review.
- [x] **SCREENSHOT** Please add a screen shot or two.
- [x] **SPECS** Please cover your changes with specs.
- [x] **READ DOCS** Please make sure you have read and understand the [Playbook Release Process](https://github.com/powerhome/playbook/wiki/Playbook-Releases)

File: playbook-website/app/javascript/site_styles/_opacity_guidelines.scss [hunk 10]
```diff
@@ -0,0 +1,48 @@
...
+  margin: none;
+  background: none;
+  border-radius: none;
+  border: 0px;
+  padding: none;
+  flex-wrap: wrap;
+  }
+}
+
+.selector {
```
Comment (jasoncypret) [inline playbook-website/app/javascript/site_styles/_opacity_guidelines.scss:17]: Lets improve this class name? What is this? is this a container of content? perhaps it is containing the opacity guidelines. Why not name it `.opacity-guideline-container {`

File: playbook-website/app/javascript/site_styles/_opacity_guidelines.scss [hunk 11]
```diff
@@ -0,0 +1,48 @@
+ul {
```
Comment (jasoncypret) [inline playbook-website/app/javascript/site_styles/_opacity_guidelines.scss:1]: Give this a class name. This will affect EVERY list on the site. Major style bleed.

File: playbook-website/app/javascript/site_styles/_opacity_guidelines.scss [hunk 12]
```diff
@@ -0,0 +1,48 @@
...
+  list-style: none;
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  margin: none;
+  background: none;
+  border-radius: none;
+  border: 0px;
+  padding: none;
+  flex-wrap: wrap;
```
Comment (jasoncypret) [inline playbook-website/app/javascript/site_styles/_opacity_guidelines.scss:13]: Indendation
```suggestion
    list-style: none;
    display: flex;
    justify-content: flex-start;
    align-items: center;
    margin: none;
    background: none;
    border-radius: none;
    border: 0px;
    padding: none;
    flex-wrap: wrap;
```

File: playbook-website/app/javascript/site_styles/_opacity_guidelines.scss [hunk 13]
```diff
@@ -0,0 +1,48 @@
...
+  }
+}
+
+.selector { 
+  .opacity_1{
```
Comment (jasoncypret) [inline playbook-website/app/javascript/site_styles/_opacity_guidelines.scss:18]:
```suggestion
  .opacity_1 {
```

File: playbook-website/app/javascript/site_styles/_opacity_guidelines.scss [hunk 14]
```diff
@@ -0,0 +1,48 @@
...
+    opacity: $opacity_1;
+  }
+  .opacity_2{
```
Comment (jasoncypret) [inline playbook-website/app/javascript/site_styles/_opacity_guidelines.scss:21]:
```suggestion
  .opacity_2 {
```
You get the picture 😉

File: playbook-website/app/javascript/site_styles/_visual_guidelines.scss [hunk 15]
```diff
@@ -41,4 +42,49 @@
...
+  .opacity_8{
+    opacity: .8;
+  }
+  .opacity_9{
+    opacity: .9;
+  }
+  .opacity_10{
+    opacity: 1;
+  }
+}
```
Comment (jasoncypret) [inline playbook-website/app/javascript/site_styles/_visual_guidelines.scss:90]:
```suggestion
```

Why is this included twice?

File: playbook-website/app/views/pages/visual_guidelines_partials/_opacity.html.erb [hunk 16]
```diff
@@ -0,0 +1,77 @@
...
+<li data-name="Data 1" data-variable="$data_1">
+<div class="pb--color_data_1 opacity_3">
+</div>
+<p class="name">Opacity_3</p>
+<p class="var">$opacity_3</p>
+</li>
+<li data-name="Data 1" data-variable="$data_1">
+<div class="pb--color_data_1 opacity_4">
+</div><p class="name">Opacity_4</p>
+<p class="var">$opacity_4</p>
```
Comment (jasoncypret) [inline playbook-website/app/views/pages/visual_guidelines_partials/_opacity.html.erb:38]: You have 0 indentations here. I can't tell what is a container. Please correct this whole file.

Tab to indent
Shift + Tab to outdent

**Review by jasoncypret — CHANGES_REQUESTED** [review]

File: playbook-website/app/javascript/site_styles/_opacity_guidelines.scss (hunk 11 above)
Comment (thestephenmarshall) [reply playbook-website/app/javascript/site_styles/_opacity_guidelines.scss:1]: Agreed. @cvargaspower specificity is of great importance any time you are working with CSS unless you are attempting to style everything in the browser view. This selector would potentially have unintended consequences as you can reference from the CSS courses.

File: playbook-website/app/javascript/site_styles/_opacity_guidelines.scss (hunk 12 above)
Comment (thestephenmarshall) [reply playbook-website/app/javascript/site_styles/_opacity_guidelines.scss:13]: @cvargaspower did you install the VSCode extension `Beautify`?

Screen Shot 2021-11-19 at 8 46 57 AM

If not, you will need to do that now. On your working files, you should right click and choose "Format Document" and these things (as mentioned above) will auto-fix.

File: playbook-website/app/javascript/site_styles/_opacity_guidelines.scss [hunk 17]
```diff
@@ -0,0 +1,48 @@
+ul {
+   
+  &.opacity-play {
```
Comment (thestephenmarshall) [inline playbook-website/app/javascript/site_styles/_opacity_guidelines.scss:3]: Not sure I like this selector name. What are we styling here? Can we be more specific? `-play` seems too generic and doesn't tell me a lot about the context.

File: playbook-website/app/views/pages/visual_guidelines_partials/_opacity.html.erb [hunk 18]
```diff
@@ -0,0 +1,74 @@
...
+      <p class="var">$opacity_10</p>
+   </li>
+</ul>
+<% react_contents = Rails.root.join("app/views/pages/code_snippets/user_jsx.txt").read %>
+<%= pb_rails("background", props: { background_color: "dark", padding: "xs", id: "code-wrapper", padding_left: "lg" }) do %>
+<div class="pb--codeCopy" data-action="toggle" data-togglable="code_example">
+   <pre class="highlight">Example in SCSS<p><code>@import "tokens/opacity";</code></p></pre>
+</div>
+<% end %>  
+<% end %>
```
Comment (thestephenmarshall) [inline playbook-website/app/views/pages/visual_guidelines_partials/_opacity.html.erb:74]:
```suggestion
<% end %>

```

The ⛔️ at the end of the file here
Screen Shot 2021-11-19 at 8 50 39 AM

indicates a new line *must* be added.

**Review by thestephenmarshall — DISMISSED** [review]
Needs work still. Let's keep on this PR.

File: playbook-website/app/javascript/site_styles/_opacity_guidelines.scss (hunk 17 above)
Comment (cvargaspower) [reply playbook-website/app/javascript/site_styles/_opacity_guidelines.scss:3]: We aren't styling anything, the way I built the opacity circles, is I used "pb--utlities-color", but it was giving it margins and borders so I used "opacity-play" to get rid of all of that. I am not sure what to name it ;u; '' Do you have a suggestion?

File: playbook-website/app/javascript/site_styles/_opacity_guidelines.scss [hunk 19]
```diff
@@ -0,0 +1,55 @@
+ul.opacity-removing-utilities {
+
+  &.opacity-removing-utilities {
```
Comment (thestephenmarshall) [inline playbook-website/app/javascript/site_styles/_opacity_guidelines.scss:3]: @cvargaspower this would translate to:

```html
<ul class="opacity-removing-utilities opacity-removing-utilities">
```

I don't know that this was your intent?

File: playbook-website/app/javascript/site_styles/_opacity_guidelines.scss [hunk 20]
```diff
@@ -0,0 +1,55 @@
...
+  }
+
+  .opacity_9 {
+    opacity: $opacity_9;
+  }
+
+  .opacity_10 {
+    opacity: $opacity_10;
+  }
+
```
Comment (thestephenmarshall) [inline playbook-website/app/javascript/site_styles/_opacity_guidelines.scss:55]:
```suggestion
  
  
```

Comment (thestephenmarshall) [reply playbook-website/app/javascript/site_styles/_opacity_guidelines.scss:55]: Not sure if this is needed now

File: playbook-website/app/javascript/site_styles/_visual_guidelines.scss [hunk 21]
```diff
@@ -1,25 +1,30 @@
...
+
+  @include break_at(breakpoint("sm")) {
     .visual_guidelines_hero {
       padding: $space_xl !important;
     }
   }
+
   .positioning-guideline {
-    & > * {
+    &>* {
```
Comment (thestephenmarshall) [inline playbook-website/app/javascript/site_styles/_visual_guidelines.scss:27]: 🤔

File: playbook-website/app/javascript/site_styles/_visual_guidelines.scss [hunk 22]
```diff
@@ -31,14 +36,96 @@
...
+    .ten {
+      margin-left: 450px;
+    }
+  }
+
+  ul.opacity-removing-utilities {
+
+    &.opacity-removing-utilities {
+      list-style: none;
+      display: flex;
```
Comment (thestephenmarshall) [inline playbook-website/app/javascript/site_styles/_visual_guidelines.scss:80]: Doesn't this exist in the other file above?

**Review by thestephenmarshall — COMMENTED** [review]
Getting there

File: playbook-website/app/javascript/site_styles/_visual_guidelines.scss (hunk 21 above)
Comment (cvargaspower) [reply playbook-website/app/javascript/site_styles/_visual_guidelines.scss:27]: That isn't me? I think Dat is doing positioning.

File: playbook-website/app/javascript/site_styles/_visual_guidelines.scss (hunk 22 above)
Comment (cvargaspower) [reply playbook-website/app/javascript/site_styles/_visual_guidelines.scss:80]: Yes, as I said in my notes: I made an opacity_guidelines.scss but it would not connect to the opacity.html.erb and only worked when I put it underneath the visual_guidelines.scss for some reason. Paul tried to help me but we both were stumped. ... maybe someone else can help.

File: playbook-website/app/javascript/site_styles/_visual_guidelines.scss (hunk 21 above)
Comment (cvargaspower) [reply playbook-website/app/javascript/site_styles/_visual_guidelines.scss:27]: I am not sure what I am supposed to do here. This was here before I made changes to the file.

File: playbook-website/app/javascript/site_styles/_visual_guidelines.scss [hunk 23]
```diff
@@ -1,6 +1,7 @@
 @import "playbook-ui/dist/tokens/spacing";
 @import "playbook-ui/dist/tokens/screen_sizes";
 @import "playbook-ui/dist/tokens/opacity";
+@import "./opacity_guidelines"
```
Comment (jasoncypret) [inline playbook-website/app/javascript/site_styles/_visual_guidelines.scss:4]: @import "opacity_guidelines"

File: playbook-website/app/javascript/site_styles/_visual_guidelines.scss [hunk 24]
```diff
@@ -35,15 +41,41 @@
...
+      margin-left: 350px;
+    }
+
+    .nine {
+      margin-left: 400px;
+    }
+
+    .ten {
+      margin-left: 450px;
+    }
```
Comment (jasoncypret) [inline playbook-website/app/javascript/site_styles/_visual_guidelines.scss:78]:
```suggestion
```

Unless you are utilizing this it has been removed from master

File: playbook-website/app/javascript/site_styles/_visual_guidelines.scss [hunk 25]
```diff
@@ -10,18 +12,22 @@
...
   &_hero {
     padding: $space_xl * 1.75 !important;
   }
+ 
   .token-wrapper {
     max-width: 1242px;
     margin: auto;
   }
-  @include break_at(breakpoint("sm")){
+
```
Comment (jasoncypret) [inline playbook-website/app/javascript/site_styles/_visual_guidelines.scss:24]:
```suggestion
```

File: playbook-website/app/views/pages/visual_guidelines.html.erb [hunk 26]
```diff
@@ -14,6 +14,7 @@
 
   <%= pb_rails("background", props: { classname: "token-wrapper", background_color: "light", padding: "xl" }) do %>
       <%= render "pages/visual_guidelines_partials/colors.html.slim" %>
+      <%= render "pages/visual_guidelines_partials/opacity.html.erb" %>
```
Comment (jasoncypret) [inline playbook-website/app/views/pages/visual_guidelines.html.erb:17]:
```suggestion
      <%= render "pages/visual_guidelines_partials/opacity" %>
```

Extension is not needed

File: playbook-website/app/views/pages/visual_guidelines_partials/_opacity.html.erb [hunk 27]
```diff
@@ -0,0 +1,75 @@
...
+      <div class="pb--color_data_1 opacity_10"></div>
+      <p class="name">Opacity_10</p>
+      <p class="var">$opacity_10</p>
+   </li>
+</ul>
+<% react_contents = Rails.root.join("app/views/pages/code_snippets/user_jsx.txt").read %>
+<%= pb_rails("background", props: { background_color: "dark", padding: "xs", id: "code-wrapper", padding_left: "lg" }) do %>
+<div class="pb--codeCopy" data-action="toggle" data-togglable="code_example">
+   <pre class="highlight">Example in SCSS<p><code>@import "tokens/opacity";</code></p></pre>
+</div>
```
Comment (jasoncypret) [inline playbook-website/app/views/pages/visual_guidelines_partials/_opacity.html.erb:72]: Check out how the other files are doing this code sample and replicate. Move the react contents variable to the pages controller as well. Use an actual code sample with an opacity token in action. See the others for example

**Review by jasoncypret — CHANGES_REQUESTED** [review]
You need to get the latest from master. I cleaned up all the files and now you have better examples of how this should be.

File: playbook-website/app/javascript/site_styles/_visual_guidelines.scss [hunk 28]
```diff
@@ -1,8 +1,17 @@
 @import "playbook-ui/dist/tokens/spacing";
 @import "playbook-ui/dist/tokens/screen_sizes";
+@import "playbook-ui/dist/tokens/opacity";
```
Comment (jasoncypret) [inline playbook-website/app/javascript/site_styles/_visual_guidelines.scss:3]: Still need this file cleaned up

File: playbook-website/app/views/pages/visual_guidelines_partials/_opacity.html.erb [hunk 29]
```diff
@@ -0,0 +1,76 @@
+<h1 aria-label="online" class="pb_title_kit_1">Opacity</h1>
+<br>
```
Comment (jasoncypret) [inline playbook-website/app/views/pages/visual_guidelines_partials/_opacity.html.erb:2]: Lets get rid of break tags, use spacing props and title kits

File: playbook-website/app/views/pages/visual_guidelines_partials/_opacity.html.erb [hunk 30]
```diff
@@ -0,0 +1,76 @@
...
+   </li>
+</ul>
+<% react_contents = Rails.root.join("app/views/pages/code_snippets/opacity_tokens.txt").read %>
+    
+<%= pb_rails("background", props: { background_color: "dark", padding: "xs", id: "code-wrapper" }) do %>
+  <%= pb_rails("caption", props: { padding_x: "md", padding_top: "md", text: "Example in Sass" }) %>
+    <div class="pb--codeCopy" data-action="toggle" data-togglable="code_example">
+      <pre class="highlight"><%= raw rouge(react_contents, "react") %></pre>
+    </div>    
+<% end %>
```
Comment (jasoncypret) [inline playbook-website/app/views/pages/visual_guidelines_partials/_opacity.html.erb:75]: Let's improve this section. Take a look at this: https://github.com/powerhome/playbook/blob/master/playbook-website/app/views/pages/visual_guidelines_partials/_positioning.html.erb#L31-L36

1. Move the `Rails.root.join("app/views/pages/code_snippets/opacity_tokens.txt").read` to the pages controller
2. Match the other styling/settings referenced in the file above.

**Review by jasoncypret — COMMENTED** [review]
I don't think my requested changes were addressed. 
1. [Your SASS file add a bunch of unused styles](https://github.com/powerhome/playbook/blob/d381b470a3e97ef3d8cbce6c55f348154495ee1c/playbook-website/app/javascript/site_styles/_visual_guidelines.scss) see [this file](https://github.com/powerhome/playbook/blob/master/playbook-website/app/views/pages/visual_guidelines_partials/_positioning.html.erb) for example. 
2. [Your erb file](https://github.com/powerhome/playbook/blob/d381b470a3e97ef3d8cbce6c55f348154495ee1c/playbook-website/app/views/pages/visual_guidelines_partials/_opacity.html.erb) doesn't match the patterns from the other files. It contains break tags instead of using spacing props and don't use kits for the title. [Here's an example](https://github.com/powerhome/playbook/blob/master/playbook-website/app/views/pages/visual_guidelines_partials/_positioning.html.erb) of how the top should look.

File: playbook/dist/playbook.css [hunk 31]
```diff
@@ -34,6 +34,8 @@ Background colors ------------------*/
     background: #F3F7FB; }
   [class^=pb_background_kit].pb_background_color_white {
     background: #FFFFFF; }
+<<<<<<< HEAD
```
Comment (jasperfurniss) [inline playbook/dist/playbook.css:37]: You gotta get rid of these first.

File: playbook/dist/playbook.css [hunk 32]
```diff
@@ -102,6 +104,7 @@ Background colors ------------------*/
     background: #9E64E9; }
   [class^=pb_background_kit].pb_background_color_insulation {
     background: #FF2229; }
+>>>>>>> 0de8ca96b833aca94a18f93ba968fad69d09a3b5
```
Comment (jasperfurniss) [inline playbook/dist/playbook.css:107]: And get rid of this.

File: playbook-website/app/views/pages/visual_guidelines_partials/_opacity.html.erb [hunk 33]
```diff
@@ -0,0 +1,70 @@
...
+      <p class="name">Opacity_10</p>
+      <p class="var">$opacity_10</p>
+   </li>
+</ul>    
+<%= pb_rails("background", props: { background_color: "dark", padding: "xs", id: "code-wrapper" }) do %>
+  <%= pb_rails("caption", props: { padding_x: "md", padding_top: "md", text: "Example in Sass" }) %>
+    <div class="pb--codeCopy" data-action="toggle" data-togglable="code_example">
+      <pre class="highlight"><%= raw rouge(@opacity_code_example, "react") %></pre>
+    </div>    
+<% end %>
```
Comment (jasperfurniss) [inline playbook-website/app/views/pages/visual_guidelines_partials/_opacity.html.erb:69]: Can you indent lines 12-63. And then also un-indent lines 66-68 by one tab.

File: playbook-website/app/javascript/site_styles/_visual_guidelines.scss (hunk 24 above)
Comment (jasperfurniss) [reply playbook-website/app/javascript/site_styles/_visual_guidelines.scss:91]: @cvargaspower This all needs to match what is on Master. You just need to commit Jason's suggestion. I think you only need really the one line of code in this file on line 3: `@import "playbook-ui/dist/tokens/opacity";`

**Review by kre8sions — APPROVED** [review]

File: playbook-website/app/javascript/site_styles/_visual_guidelines.scss [hunk 34]
```diff
@@ -19,13 +28,69 @@
...
+
+    .nine {
+      margin-left: 400px;
+    }
+
+    .ten {
+      margin-left: 450px;
+    }
+  }
+
```
Comment (jasoncypret) [inline playbook-website/app/javascript/site_styles/_visual_guidelines.scss:93]:
```suggestion
```

**Review by jasoncypret — CHANGES_REQUESTED** [review]
I marked what needs to be done to get this PR in. You should just have to accept and make sure you don't have any merge conflicts

**Comment by thestephenmarshall** [comment]
@nickamantia this one has been very long in the tooth and @jasoncypret needs to re-review here in order to get it moving.

**Comment by jasoncypret** [comment]
@nickamantia I'm having trouble viewing the Milano deploy.

image

**Review by jasoncypret — CHANGES_REQUESTED** [review]
I don't think this is related to yours but this section is now missing it's background. 
image

Can you fix it up with this PR?

**Comment by nickamantia** [comment]
Line Height background fix:

Screen Shot 2022-02-16 at 11 55 34 AM
